torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -286,6 +287,48 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        prim::BroadcastSizes,
+        [](const Node* node) -> Operation {
+          size_t num_inputs = node->inputs().size();
+          return [num_inputs](Stack& stack) {
+            std::vector<int64_t> size;
```

Comment:
might be faster with SmallVector?

Reply:
I have to push this on the stack later, so it will need to end up as a regular vector anyway 😕